slurm-spank-talamini.spec

```diff
@@ -39,7 +38,7 @@ install -d %{buildroot}%{_libexecdir}/slurm
 install -d %{buildroot}%{_sysconfdir}/slurm/plugstack.conf.d
 install -m 755 env-test.so %{buildroot}%{_libdir}/slurm/
 install -m 755 pbs_nodefile.so %{buildroot}%{_libdir}/slurm/
-install -m 755 generate_pbs_nodefile.pl %{buildroot}%{_libexecdir}/slurm/generate_pbs_nodefile
+install -m 755 generate_pbs_nodefile.py %{buildroot}%{_libexecdir}/slurm/generate_pbs_nodefile
```

Comment:
is this the only purpose of the talamini? IF (so a big if) during profile the slurm variables already exist, i am ok with generating this during profile